spec/spec_helper.rb

```diff
@@ -14,23 +14,30 @@ def define_presenter(name = "ShowPresenter", &block)
     presenter_class
   end
 
-  def render(source, locals = {}, presenter_class = nil, &block)
-    if presenter_class.nil?
-      unless defined?(ShowPresenter)
-        define_presenter("ShowPresenter")
-      end
+  def render(source, options = {}, &block)
```

Comment:
The reason I changed it was because I felt that you would end up with two hash parameters, with one being the locals and one being the options, and I didn't like that; so I switched it around and made the locals a key pair in the options hash.